inttest/Makefile

```diff
@@ -17,23 +18,30 @@ bin:
 bin/sonobuoy: | bin
 	$(curl) $(sonobuoy_url) | tar -C bin/ -zxv $(notdir $@)
 
+kine.conf:
```

Comment:
shouldn't this be `k0s.yaml` (or `k0s.conf`)? since it's not actually kine's config, but k0s' config.

Reply:
We could use `k0s.yaml` or `k0s.conf` but this is config for check-basic testing kine specifically. We could other configs for other tests in future too.

Reply:
I renamed it to `k0s-kine.conf`